app/helpers/application_helper.rb

```diff
@@ -5,4 +5,8 @@ def js_prefix
     prefix = 'js' + Digest::MD5.hexdigest(SecureRandom.hex + Time.now.strftime('%Y%m%d%H%M%S'))
     prefix
   end
+
+  def nl2br(str)
```

Comment:
__[reek]__
* [UtilityFunction] doesn't depend on instance state (maybe move it to another class?) [:ref](https://github.com/troessner/reek/blob/master/docs/Utility-Function.md)

<!-- Comment by [SideCI](https://sideci.com) -->